src/python/pants/backend/codegen/thrift/java/java_thrift_library.py

```diff
@@ -84,7 +84,6 @@ def default_java_namespace(self):
   def include_paths(self):
     return self._include_paths
 
-  # TODO(Eric Ayers) As of 2/5/2015 this call is DEPRECATED and should be removed soon
   @property
```

Comment:
I see what you mean. The usecase in jvm_target can't work unless the cycle were to be broken.

Marker mixins are classes that have no behavior, but mark a class as being a particular type. It's a python multiple inheritance thing. eg

```
class ThriftMixin(object):
  pass

class Foo(ThriftMixin, Bar):
  pass
```